cli/execute.py

```diff
@@ -19,32 +21,6 @@
     CONF.get('defaults', 'playbooks')), "Bad path to playbooks"
```

Comment:
Why do we need that assert here? It looks like we are forcing user to use "playbooks" folder. In that case we don't need configuration option for playbooks.